encord/objects/common.py

```diff
@@ -363,6 +582,10 @@ def add_nested_option(
         """
         return _add_attribute(self.nested_options, cls, name, self.uid, local_uid, feature_node_hash, required)
 
+    def __hash__(self):
```

Comment:
These were added so these objects can be added into a `set`. I can also add an `__eq__` operator if you feel like it would make the object more complete.